mix.exs

```diff
@@ -52,9 +52,9 @@ defmodule OpentelemetryLiveView.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:opentelemetry_api, "~> 1.0.0-rc.3"},
-      {:opentelemetry, "~> 1.0.0-rc.3", only: [:test]},
-      {:opentelemetry_telemetry, "~> 1.0.0-beta"},
+      {:opentelemetry_api, "~> 1.0.0-rc.4.1"},
+      {:opentelemetry, "~> 1.0.0-rc.4", only: [:test]},
+      {:opentelemetry_telemetry, "~> 1.0.0-beta.6.1"},
```

Comment:
There have been some updates :) `opentelemetry_api` and `opentelemetry` are `1.0.0` and `opentelemetry_telemetry` is `1.0.0-beta.7`

Reply:
Going to update this in a separate PR.